db/migrate/20250414121956_migrate_ovirt_resources.rb

```diff
@@ -0,0 +1,24 @@
+class MigrateOvirtResources < ActiveRecord::Migration[7.0]
+  def up
+    ovirt_ids = ComputeResource.unscoped.where(type: 'Foreman::Model::Ovirt').pluck(:id)
+
+    ovirt_ids.each do |cr_id|
+      Rails.logger.debug "Processing data for compute resource ID #{cr_id}."
+
+      Host.unscoped.where(compute_resource_id: cr_id).find_each do |host|
+        Rails.logger.debug "Disassociating host '#{host.name}' from compute resource ID #{cr_id}."
+        host.disassociate!
```

Comment:
@stejskalleos did you try it without `Foreman::Model::Ovirt` class? I think it would not be able to load the `host` object, since it will try to set the compute_resource property to a non-existing model.

Reply:
Maybe since `disassociate` just updates two properties: https://github.com/theforeman/foreman/blob/62b3353e6b557c70f4e1e51f6d2b0de48001c119/app/models/host/managed.rb#L589
we can bulk-update those properties directly from the migration?